docs/Reference.md

```diff
@@ -751,6 +783,7 @@ lockIdentifier
 mainDocumentURL (iOS only)
 navigationType (iOS only)
 isTopFrame (iOS only)
+hasTargetFrame (iOS only)
```

Comment:
I'm wondering if perhaps this could be explained in the doc a little bit? I realize the other props aren't well-explained either... but `hasTargetFrame` seems to me like it could be especially confusing... for example, I assume that it would always be `true` if a handler is provided for `onOpenWindow`, yes?

Reply:
> in Practice, if OnOpenWindow is defined, it should always be false because otherwise OnShouldStartLoadWithRequest would not even be called (OnOpenWindow is called instead)

I thought the opposite was the case. `if (_onOpenWindow && !hasTargetFrame) {` looks like would only call `onOpenWindow` if `hasTargetFrame` is `false`, leaving `onShouldStartLoadWithRequest` to be called with `true`...? Am I misreading this?

> It is false if the webView's navigation trigger the current frame

`false` isn't the `target="_blank"` scenario?

> Do you know what is the expected format for documenting this kind of props? I don't find any similar exemple in the existing doc. Should I just add it in a paragraph after the props list?

I think a paragraph after the props list would be a fine way to do it. `onError` appears to have an example for this.

Reply:
You're right, I answered too fast based on my memories but said the inverse of the real behaviour. Sorry for the confusion.

I added the doc here: https://github.com/react-native-webview/react-native-webview/compare/09f909649f0b42f85306f47b3b1654c94bd0054e..3ba2f992c1a6349e35796a50f13bcdf6ba80df60

Tell me if it is not clear enough

Reply:
Looks good, thanks!